component-library/src/translations/en-US.json

```diff
@@ -35,6 +35,8 @@
   },
   "OPPOSITE_LANGUAGE": "Français",
   "Header_URL": "https://www.canada.ca/etc/designs/canada/wet-boew/assets/sig-blk-en.svg",
+  "HEADER_IMG_LINK": "https://www.canada.ca/en.html",
```

Comment:
Can't do this, since the library doesn't automatically combine its translate files with the project files